easy-tutorial-step1/README.md

```diff
@@ -26,29 +40,27 @@ First thing is then to install the latest 2205 commerce version. Follow the offi
 4. Run `ant initialize -Dtenant=master`
 5. Start SAP Commerce Cloud server with `<HYBRIS_HOME>/bin/platform/hybrisserver.sh`
 
-### Configure easy repositories
-Next you'll have to configure the git repository where the tutorial extension is stored. If you want to play around locally with the extension we suggest you to create a local easy repository that will simplify and speed up your development.
+### Configure local Easy repository
```

Comment:
Link this to: https://sap.github.io/easy-extension-framework/configuring-an-easy-repository-in-backoffice/